### PR TITLE
inject DQT creds for staging and prod

### PR DESCRIPTION
**Pull request by asmega**
### Context

- NPQ registration staging connects to ecf staging
- NPQ registration production connects to ecf production
- These NPQ environments require DQT client configured and functioning in ECF
- Before this is merged someone will need to add the credentials to the encrypted yaml file for staging and prod
- We do not have a test environment for DQT yet, so for now can we reuse the dev credentials for DQT in staging please

### Changes proposed in this pull request

- Load DQT creds from encrypted yaml

### Guidance to review

- staging and prod credentials are in place first

### Testing

- load credentials for staging and prod
- should be able to make DQT calls

### Review Checks
- [x] All pages have automated accessibility checks via cypress
- [x] All pages have visual tests via cypress + percy
- [x] All `School` queries are correctly scoped - `eligible` when they need to be

### How can I view this in a review app?

- probably not possible

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -58,6 +58,10 @@
   config.gias_api_user = ENV["GIAS_API_USER"]
   config.gias_api_password = Rails.application.credentials.GIAS_API_PASSWORD
 
+  config.dqt_client_api_key = Rails.application.credentials.DQT_CLIENT_API_KEY
```
Comment (jamesrose) [inline config/environments/production.rb:61]: Could do this in application.rb:

```
config.dqt_client_api_key = Rails.application.credentials.DQT_CLIENT_API_KEY || ENV["DQT_CLIENT_API_KEY"]
```

and so on.

Comment (asmega) [reply config/environments/production.rb:61]: I've just copy pasted the current convention from `deployed_development`. Those working more closely on ECF may have an opinion on how they want it.

**Review by kicferk1 — COMMENTED** [review]
I would expect a change to the encrypted credentials files too. Do they already contain the keys? @ethanmills

**Comment by asmega** [comment]
> I would expect a change to the encrypted credentials files too. Do they already contain the keys? @ethanmills

Perfect segue. So since I don't have these creds can someone add them to this PR or raise and merge another PR before this.

**Comment by ethanmills** [comment]
Just added creds, detail in commit message

**Review by kicferk1 — APPROVED** [review]

**Comment by asmega** [comment]
awesome. thanks all!

**Pull request merged**